src/main/java/com/patchworkmc/patch/StringConstantRemapper.java

```diff
@@ -7,16 +7,16 @@
 
 import com.patchworkmc.Patchwork;
 import com.patchworkmc.mapping.remapper.AmbiguousMappingException;
-import com.patchworkmc.mapping.remapper.NaiveRemapper;
+import com.patchworkmc.mapping.remapper.PatchworkRemapper;
 
 /**
  * Remaps Strings into Intermediary. Necessary for things like reflection and ObfuscationRemapperHelper,
  * unless we put SRG and tiny-remapper on the classpath at runtime.
  */
 public class StringConstantRemapper extends ClassVisitor {
-	private NaiveRemapper remapper;
+	private PatchworkRemapper remapper;
 
-	public StringConstantRemapper(ClassVisitor classVisitor, NaiveRemapper remapper) {
+	public StringConstantRemapper(ClassVisitor classVisitor, PatchworkRemapper remapper) {
```

Comment:
Doesn't it make sense to just take `PatchworkRemapper.Naive` as an argument since that's all that this class uses?